### PR TITLE
Feature/add tours

### PR DESCRIPTION
**Pull request by rickstylz01**

### CODE REVIEW

File: src/routes/emailRoutes.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
 const express = require('express');
```
Comment (seyiogunjuyigbe) [inline src/routes/emailRoutes.js:1]: is there a reason for this route?

File: src/routes/userRoutes.js [hunk 2]
```diff
@@ -24,4 +24,4 @@ router.put('/user/id', User.update);
 // DELETE
 router.delete('/user/:id', User.destroy);
```
Comment (seyiogunjuyigbe) [inline src/routes/userRoutes.js:25]: this route is not needed

File: src/services/bookTourService.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+// const Booking = require('../models/booking');
```
Comment (seyiogunjuyigbe) [inline src/services/bookTourService.js:1]: you can just delete the file instead

File: src/controllers/userController.js [hunk 4]
```diff
@@ -12,6 +12,7 @@ exports.index = async function (req, res) {
 // @route POST api/user
 // @desc Add a new user
 // @access Public
+// TODO: abstract and reuse in authController
 exports.store = async (req, res) => {
```
Comment (seyiogunjuyigbe) [inline src/controllers/userController.js:16]: this will not be necessary, as well as the route that uses this function

**Review by seyiogunjuyigbe — CHANGES_REQUESTED** [review]

File: src/routes/userRoutes.js (hunk 2 above)
Comment (rickstylz01) [reply src/routes/userRoutes.js:25]: OK.  I will also delete the destroy controller afterwards.

**Pull request merged**